tests/unit/initializers/export-application-global-test.js

```diff
@@ -27,17 +27,19 @@ module('ExportApplicationGlobalInitializer', {
 
 test('it sets the application on window with the classified modulePrefix', function(assert) {
   config.modulePrefix = 'foo';
-  initialize(null, 'blazorz');
+  var app = { reopen: function(){} };
+  initialize(null, app);
 
-  assert.equal(window.Foo, 'blazorz');
+  assert.equal(window.Foo, app);
 });
 
 test('it does not set the global unless exportApplicationGlobal is true', function(assert) {
   config.modulePrefix = 'foo';
   config.exportApplicationGlobal = false;
```

Comment:
now we are: https://github.com/ember-cli/ember-export-application-global/commit/9c12bfe21cc79f164b32b96872794d13bde40e01